app/src/main/java/com/codepath/chefster/client/FirebaseClient.java

```diff
@@ -81,4 +82,10 @@ public void uploadNewDataToDatabase(){
             mDatabase.child("dishes").child(String.valueOf(dish.getUid())).setValue(dish);
         }
     }
+
+    //upload the reviews to database
+    public void uploadDishReviewsToDatabase(Review review, Integer dishId){
```

Comment:
we will have reviews in my next PR with exactly the same signature.
you can remove it because there's something missing and it won't work.